Updated documentation for influxdb-integration (#25823)
home-assistant/home-assistant.github.io: source/_integrations/influxdb.markdown

    api_version: 2
    organization: RANDOM_16_DIGIT_HEX_ID
    token: GENERATED_AUTH_TOKEN

>     bucket: MYBUCKET 

Reviewer: ```suggestion
    bucket: "MYBUCKET"
```

        range_start: "-1d"
```

> The influxdb sensor for `api_version: 2` *rewrites* the flux-query before it is sent to the database. As a minimal, the bucket and range are prefixed and a '|> limit(n: 1)' postfixed. 

Reviewer: ```suggestion
The InfluxDB sensor for `api_version: 2` *rewrites* the flux query before it is sent to the database. As a minimal, the bucket and range are prefixed and a `|> limit(n: 1)` is postfixed. 
```

> If you do not specify the bucket as a config, you can write the exact query (including 'from(bucket: "MYBUCKET")'-statements etc).

Reviewer: ```suggestion
If you do not specify the bucket as a config, you can write the exact query (including `from(bucket: "MYBUCKET")`-statements, etc).
```

```yaml
# Example configuration.yaml entry, recommended to validate the query in InfluxDB UI
sensor: 
  - platform: influxdb
    api_version: 2

>     organization: RANDOM_16_DIGIT_HEX_ID

Reviewer: ```suggestion
    organization: "RANDOM_16_DIGIT_HEX_ID"
```

>     token: GENERATED_AUTH_TOKEN

Reviewer: ```suggestion
    token: "GENERATED_AUTH_TOKEN"
```

    api_version: 2
    token: GENERATED_AUTH_TOKEN
    organization: RANDOM_16_DIGIT_HEX_ID
    queries_flux:
      - range_start: "-1d"
        name: "How long have I been here"

>         bucket: BUCKET_NAME

Reviewer: ```suggestion
        bucket: "BUCKET_NAME"
```
